app/models/grda_warehouse/tasks/push_clients_to_cas.rb

```diff
@@ -89,6 +89,9 @@ def sync!
               project_client.enrolled_in_rrh = client.enrolled_in_rrh(enrollments)
               project_client.enrolled_in_psh = client.enrolled_in_psh(enrollments)
               project_client.enrolled_in_ph = client.enrolled_in_ph(enrollments)
+              project_client.ongoing_es_enrollments = client.processed_service_history&.cohorts_ongoing_enrollments_es&.map { |e| e['project_name'] + ':' + e['date'] }.presence
+              project_client.ongoing_so_enrollments = client.processed_service_history&.cohorts_ongoing_enrollments_so&.map { |e| e['project_name'] + ':' + e['date'] }.presence
```

Comment:
Should these have a space to the right of the colon?  Or maybe the date should be labeled somehow?

Reply:
They should, I was trying to match how the project/date was shown elsewhere, but missed the space. It also looks like I missed some date formatting. With that said, I am open to suggestions on better ways to display the information

Reply:
Let's go with `: ` for now.  Thanks!

Reply:
I've pushed a new commit to address the ":" and date format